### PR TITLE
docs: done chapter 1-1.mdx issue #51

### PR DESCRIPTION
**Pull request by victorescosta**
issue #51

### CODE REVIEW

**Review by lewtun — COMMENTED** [review]
Thank you for this nice translation @victorescosta 🥳 ! We just need to include this section in the `_toctree.yml` file, e.g. please add this:

```
- title: 1. Transformer models # <= Translate this
  sections:
  - local: chapter1/1
    title: Introduction # <= Translate this
```

**Comment by lewtun** [comment]
Hey @victorescosta just checking if you'd like any help on adding the `_toctree.yml` file to this PR? 

It's better to include it in this PR instead of #102 so we can test the website builds in the CI. Happy to help if you need it!

**Comment by victorescosta** [comment]
> Hey @victorescosta just checking if you'd like any help on adding the `_toctree.yml` file to this PR?
> 
> It's better to include it in this PR instead of #102 so we can test the website builds in the CI. Happy to help if you need it!

Hi @lewtun , i'm sorry i'm bit busy this week but in the weekend i'll do the changes, is it ok? I'm always open to be helped if it's needed or if we have a specific time to finish this task :)

**Comment by lewtun** [comment]
> Hi @lewtun , i'm sorry i'm bit busy this week but in the weekend i'll do the changes, is it ok? I'm always open to be helped if it's needed or if we have a specific time to finish this task :)

No rush, just wanted to check if you were stuck of not :) Looking forward to seeing the fix when you're ready!

**Comment by victorescosta** [comment]
All done, sir! @lewtun. Let's move on!

File: chapters/pt/_toctree.yml [hunk 1]
```diff
@@ -2,6 +2,31 @@
   sections:
   - local: chapter0/1
     title: Introdução
+
+- title: 1. Modelos Transformers
+  sections:
+  - local: chapter1/1
+    title: Introdução
+  - local: chapter1/2
```
Comment (lewtun) [inline chapters/pt/_toctree.yml:10]: We should only include in the table of contents the sections that have a corresponding .mdx file. So in this case, everything from this line and below can be removed :)

**Review by lewtun — COMMENTED** [review]
Thanks for iterating on this @victorescosta! There's just a small fix needed in the table of contents, but otherwise this is looking great 🥳

File: chapters/pt/_toctree.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@
   sections:
   - local: chapter0/1
     title: Introdução
+
```
Comment (lewtun) [inline chapters/pt/_toctree.yml:5]: I think you just need to add this to enable the content to render on the website. I'm not sure if my translation is an good, so feel free to change it :)

```suggestion

- title: 1. Modelos de transformadores
  sections:
  - local: chapter1/1
    title: Introdução
    
```

**Comment by victorescosta** [comment]
Thank you so much for helping me @lewtun. I think that everything is ok now, right? 🕺

**Pull request merged**